BUILDING.md

```diff
@@ -1,152 +1,147 @@
 ## BUILD INSTRUCTIONS AND REQUIREMENTS
 
-DB Browser for SQLite requires Qt as well as SQLite. For more information on Qt
-please consult http://www.qt.io and for SQLite please see https://sqlite.org/.
+DB Browser for SQLite requires Qt as well as SQLite.<br>
+For more information on Qt please consult http://www.qt.io and for SQLite please see https://sqlite.org/.
 
 Please note that all versions after 3.12.1 will require:
 * A C++ compiler with support for C++14 or later
-
-All versions after 3.9.1 will require:
-* Qt 5.5 or later, however we advise you to use 5.7 or later
-* A C++ compiler with support for C++11 or later
+* Qt 5.15.9 later
 
 Without these or with older versions you won't be able to compile DB Browser for
-Sqlite anymore. This applies to all platforms. However, most likely you won't
+SQLite anymore.<br>This applies to all platforms. However, most likely you won't
 have to worry about these as most systems meet these requirements today.
 
-If you have any chance, please use Qt 5.7 or any later version. Even though Qt
+If you have any chance, please use Qt 5.7 or any later version.<br>Even though Qt
```

Comment:
"If you have any chance, ..." should probably be "If you can, ..."

Reply:
Resolve via [c54aa27](https://github.com/sqlitebrowser/sqlitebrowser/pull/3485/commits/c54aa27024f53c735ab74e34e4b85fcd7dd8c4ec)